src/main/java/core/basesyntax/service/strategy/AddingToBalanceService.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.strategy;
+
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.exeption.FruitShopExeption;
+import core.basesyntax.service.FruitService;
+
+public class AddingToBalanceService extends FruitService {
```

Comment:
you should have OperationHandler as a interface for strategy. 
4 implementation for this handler 
and additional interface with implementation OperationStrategy, that should one method to get exact operation from strategy and contain map with operation type as key and implementation of strategy as value. 
Also, strategy should be separate package from service